tests/eval/bad/gold/list_to_map.scilexp.gold

```diff
@@ -1,7 +1,7 @@
 {
   "errors": [
     {
-      "error_message": "Syntax error, state number 36",
+      "error_message": "Syntax error, state number 31",
```

Comment:
Same suggestions as for the typechecker test.

Reply:
Same comment as for the typechecker test. :-)